doc/release-notes/10494-payara-upgrade.md

```diff
@@ -0,0 +1,6 @@
+# Upgrade Payara to v6.2024.4
+
+With this version of Dataverse, we encourage you to upgrade to version 6.2024.4.
+This will address security issues accumulated since the release of 6.2023.8, which was required since Dataverse release 6.0.
+
+If you are using GDCC containers, this upgrade is included when pulling new release images.
```

Comment:
Should note that the release instructions need to add the standard Payara update steps. With this one, I think copying the domain works fine, i.e. like the instructions for  Dataverse 5.6 rather than from our 5->6 transition.

Reply:
As requested, I added the instructions. Feel free to take a look.